src/Sylius/Bundle/UiBundle/Resources/private/js/sylius-product-attributes.js

```diff
@@ -65,6 +65,7 @@ const setAttributeChoiceListener = function setAttributeChoiceListener() {
   const $attributeChoice = $('#attributeChoice');
   $attributeChoice.find('button').on('click', (event) => {
     event.preventDefault();
+    $('form').addClass('loading');
```

Comment:
1. It should not be a content of this PR 
2. It should be already fixed https://github.com/Sylius/Sylius/pull/9796 :)